packaging/windows/BUILD.txt

```diff
@@ -0,0 +1,120 @@
+    PATH=$PATH:/usr/share/java/fop
+
+    #https://www.ict.griffith.edu.au/anthony/info/graphics/imagemagick.hints
+    export MAGICK_HOME="$PREFIX/bin"
+    export MAGICK_CONFIGURE_PATH="$PREFIX/etc/ImageMagick-7"
+    export MAGICK_CODER_MODULE_PATH="$PREFIX/lib/ImageMagick-7.0.3/modules-Q16HDRI/coders"
+
+    execute the following command to actviate  profile changes 
+    $. .bash_profile
+
+    Clone darktable git repository (in this example into ~/src):
+        $ mkdir ~/src
+        $ cd ~/src
+        $ git clone git://github.com/darktable-org/darktable.git
+    
+    Finally build and install darktable:
+        $ mkdir darktablebuild
+        $ cd darktablebuild
+        $ cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DPROJECT_VERSION=2.0.6 -DCMAKE_INSTALL_PREFIX=/opt/darktable ../darktable
+        $ cmake --build .
+        $ cmake --build . --target install
+
+        After this darktable will be installed in /opt/darktable directory and can be started by typing /opt/darktable/bin/darktable.exe in MSYS2 MINGW64 terminal.
+
+    For building the usermanual
+        Add -DBUILD_USERMANUAL=ON flag to the cmake command line
+        
+2). Download and install gtk-bundle for Windows and install it
+    https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer
+    Pls note: you need to run the installer as admin to run the cache generation
+
+    $ gdk-pixbuf-query-loaders --update-cache 
+
+3). TODO: Run install image creating script, it should create darktable-<VERSION>.msi in current (packaging/windows) directory:
+     $ cmake --build . --target package
+
+ISSUES - FIXED
+    * darktablerc config file  is being generated with Windows style line endings ( CRLF \r\n) , and config file reading functions are expecting a UNIX style line ending (only CR \n)
+    same problem with strip_semicolons_from_keymap function
+    * Reading system parameters (memory/CPU) at init - done
+    * Finding OpenCL library - done
+    * Replacing OpenCL symlinks with a filename/MD5tag combination (symlinks require admin priviledges on windows)
+    * Loading libraries / finding plugin library
+    * Removing ws2_32 and msvcrt as not needed
+    * Added LUA (needs a special old, 5.2 version of the package)
+    * Added SLIDESHOW
+    * Added OPENMAP Support
+    * Added LIBGPHOTO (camera support)
+    * Addes statvfs for Windows
+    * Added getrlimit/setrlimit using https://msdn.microsoft.com/ko-kr/library/ms811896.aspx#ucmgch09_topic9
+    * Added dt_print_mem_usage for Windows
+
+TODO LIST
+    Port lua select() in function read_cb: https://msdn.microsoft.com/en-us/library/aa363858(VS.85).aspx ?
+    Port COLORD or find a way to replace it
+    Port CUPS or find a way to replace it
+    Port FLICKCURL
+    lensfun database should be deployed to Appdata/local/lensfun (no version_1 subfolder), alternatively under darktable/share at install
+    right alignment of views at the toolbar (_lib_view_toolbox_add ?)
```

Comment:
Seems to me a GUI issue. So I have now all the modules, like tethering, map, slideshow, but instead of showing them aligned to the right, as always, it adds a dropbox called "other", and then you can select the modules. COuld not figure out yet why. See the two screenshots below
![capture01](https://cloud.githubusercontent.com/assets/7863452/19739070/e6d03c06-9bb9-11e6-81de-a5336848704e.PNG)
![capture2](https://cloud.githubusercontent.com/assets/7863452/19739069/e6cf46fc-9bb9-11e6-818b-39de52812030.PNG)